Source/Blazorise/Attributes/NumericAttribute.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using Microsoft.AspNetCore.Components;
+
+namespace Blazorise;
+
+/// <summary>
+/// Configures a numeric field.
```

Comment:
More descriptive comment.

eg. `Represents an attribute that can be applied to numeric properties or fields to specify additional metadata.`